### ISSUE TITLE
[Rule Proposal] valid-promise-race

### ISSUE THREAD

**Issue by macklinu** [xjamundx/eslint-plugin-promise#149]
### Description

Inspired by https://twitter.com/kentcdodds/status/1049808195030462464

Would check to ensure `Promise.race()` receives one argument.

```js
// valid
Promise.race([promise1, promise2])

// invalid
Promise.race(promise1, promise2)
```
